src/main/webapp/itempagescript.js

```diff
@@ -47,11 +41,13 @@ function getItemId() {
  * Sends comment data and ItemId to Servlet
  */
 async function sendFormData() {
+  const comment = $('#comment').val();
   const itemId = getItemId();
-  const comment = document.getElementById('comment');
-  fetch(
-    `/itempagedata?=${itemId}`,
-    { method: 'post', body: JSON.stringify(comment) })
+  $.post('/itempagedata', { comment: comment, itemId: itemId }).done(function() {
+    window.location.reload();
```

Comment:
Does the page really need to reload to submit a new comment or can ItemPageServlet redirect to the page by itself?

Reply:
If I don't, it'll accept the same comment multiple times. The servlet code doesn't redirect it.